sormas-api/src/main/java/de/symeda/sormas/api/utils/HtmlHelper.java

```diff
@@ -1,35 +1,53 @@
+/*******************************************************************************
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2018 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *******************************************************************************/
 package de.symeda.sormas.api.utils;
 
-import org.apache.commons.text.StringEscapeUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
 
+// This class provides general XSS-Prevention methods using Jsoup.clean
 public class HtmlHelper {
 
-	// unescape specific tags (<b>,<i>,<br>,<li>,<ul>) escaped using StringEscapeUtils.escapeHtml4(String)
-	public static String unescapeBasicTags(String escapedString) {
-		String res = escapedString;
+	public static final Whitelist EventActionWhitelist =
+		Whitelist.relaxed().addTags("hr", "font").addAttributes("font", "size", "face", "color").addAttributes("div", "align");
 
-		// <b> Tags
-		res = res.replaceAll("&lt;b&gt;", "<b>");
-		res = res.replaceAll("&lt;/b&gt;", "</b>");
-		// <i> Tags
-		res = res.replaceAll("&lt;i&gt;", "<i>");
-		res = res.replaceAll("&lt;/i&gt;", "</i>");
-		// <ul> Tags
-		res = res.replaceAll("&lt;ul&gt;", "<ul>");
-		res = res.replaceAll("&lt;/ul&gt;", "</ul>");
-		// <li> Tags
-		res = res.replaceAll("&lt;li&gt;", "<li>");
-		res = res.replaceAll("&lt;/li&gt;", "</li>");
-		// <br> Tags
-		res = res.replaceAll("&lt;br&gt;", "<br>");
-		res = res.replaceAll("&lt;/br&gt;", "<br>");
-		res = res.replaceAll("&lt;br/&gt;", "<br>");
+	public static String cleanHtml(String string) {
+		if (string == null)
```

Comment:
We're using brackets even for one-line statements; please refactor your code accordingly; also, you could use a shorter syntax for the return statements in this class (e.g. `return string == null ? "" : Jsoup.clean(string, Whitelist.none());`